Flaky integration test test_tracking_integration.py::test_tracking[using_job_queue-ee_failing_poly_smoother]

**Describe the bug**
`FAILED tests/integration_tests/status/test_tracking_integration.py::test_tracking[using_job_queue-ee_failing_poly_smoother] - Failed: Timeout >360.0s` (https://github.com/equinor/ert/actions/runs/8595749537/job/23551154574?pr=7562)

**To reproduce**
Steps to reproduce the behaviour:
1. `pip install ert`
2. `ert gui my_config.ert`
3. Run experiment (IES/Smoother/ESMDA/Test)
4. …

**Expected behaviour**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Environment**
 - OS: macos-latest
 - ERT/Komodo release: main
 - Python version: 3.12
 - Remote/HPC execution involved: Github Action Runner

**Additional context**
Add any other context about the problem here.

There seem to be events missing. One of the realizations pass, but the other one is still marked with running when the FinishEvent it sent. 
```FAILED tests/integration_tests/status/test_tracking_integration.py::test_tracking[using_scheduler-ee_poly_smoother-28-40] - AssertionError: Failed too many realizations '[RealizationSnapshot(status='Running', active=True, start_time=datetime.datetime(2024, 4, 8, 12, 19, 48, 488116, tzinfo=tzutc()), end_time=None, forward_models={})]'```

> There seem to be events missing. One of the realizations pass, but the other one is still marked with running when the FinishEvent it sent. `FAILED tests/integration_tests/status/test_tracking_integration.py::test_tracking[using_scheduler-ee_poly_smoother-28-40] - AssertionError: Failed too many realizations '[RealizationSnapshot(status='Running', active=True, start_time=datetime.datetime(2024, 4, 8, 12, 19, 48, 488116, tzinfo=tzutc()), end_time=None, forward_models={})]'`

This might have been fixed in the meanwhile by 4ababea

--- a/src/ert/async_utils.py
+++ b/src/ert/async_utils.py
@@ -19,7 +19,7 @@ def new_event_loop() -> asyncio.AbstractEventLoop:
 
 def get_event_loop() -> asyncio.AbstractEventLoop:
     try:
-        return asyncio.get_event_loop()
+        return asyncio.get_running_loop()
     except RuntimeError:
         asyncio.set_event_loop(new_event_loop())
         return asyncio.get_event_loop()

